realm/realm-jni/src/io_realm_internal_LinkView.cpp

```diff
@@ -193,11 +193,24 @@ JNIEXPORT jlong JNICALL Java_io_realm_internal_LinkView_nativeFind
 }
 
 JNIEXPORT void JNICALL Java_io_realm_internal_LinkView_nativeRemoveAllTargetRows
-  (JNIEnv *env, jobject, jlong nativeLinkViewPtr) {
+  (JNIEnv *env, jobject, jlong nativeLinkViewPtr)
+{
     TR_ENTER_PTR(nativeLinkViewPtr)
     try {
         LinkView *lv = LV(nativeLinkViewPtr);
         lv->remove_all_target_rows();
     } CATCH_STD()
- }
+}
+
+JNIEXPORT jlong JNICALL Java_io_realm_internal_LinkView_nativeGetTargetTable
+  (JNIEnv *, jobject, jlong nativeLinkViewPtr)
+{
+    TR_ENTER_PTR(nativeLinkViewPtr)
+
+    LinkView *lv = LV(nativeLinkViewPtr);
+    Table* pTable = &(lv->get_target_table());
+    LangBindHelper::bind_table_ptr(pTable);
+
+    return (jlong)pTable;
```

Comment:
I'm trying be cautious, but wouldn't it be better to use `return reinterpret_cast<jlong>(pTable);` since the returned value would be re-casted to `Table*` again?